data/data/bootstrap/vsphere/files/etc/NetworkManager/dispatcher.d/30-local-dns-prepender

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
how is this handled in the  control-plane? Using a bash script like this or something else?

Reply:
MCO installs a more sophisticated prepender - https://github.com/openshift/machine-config-operator/blob/master/templates/master/00-master/vsphere/files/NetworkManager-resolv-prepender.yaml, but its using the same idea